build.sbt

```diff
@@ -1,14 +1,14 @@
 val runScripted = taskKey[Unit]("Run supported sbt scripted tests")
 
-val scalaJSVersion = sys.env.getOrElse("SCALAJS_VERSION", "1.0.0")
+val scalaJSVersion = sys.env.getOrElse("SCALAJS_VERSION", "1.1.0")
```

Comment:
This should stay 1.0.0. Same for the versions in .travis.yml. Otherwise effectively we're still building against 1.1.0.

Reply:
Could it be 1.0.1?